### ISSUE TITLE
Calling +[UIImage imageNamed:] with nil name

### ISSUE THREAD

**Issue by keith** [Instabug/Instabug-iOS#279]
### Steps to Reproduce the Problem

Trigger showing Instabug

### Expected Behavior

No logs

### Actual Behavior

You see these logs in the console:

```
... [framework] CUICatalog: Invalid asset name supplied: '(null)'
... [framework] CUICatalog: Invalid asset name supplied: '(null)'
```

This appears to be caused by calling `+[UIImage imageNamed:]` with a nil string. You can catch Instabug doing this by setting the breakpoint outlined here: https://stackoverflow.com/a/30071894/902968

Here's the stacktrace causing this:

![image](https://user-images.githubusercontent.com/283886/43597783-f4bf0700-9637-11e8-9b5d-f42d87e27312.png)

There doesn't seem to be any user impact.

### SDK Version

8.0.7

### iOS Version

11.4.1

### Device Type

Simulator

**Comment by Kmohamed** [comment]
Are you adding Instabug bundle to the project directory ?

**Comment by keith** [comment]
Yep, the bundle is in `Foo.app/InstabugResources.bundle`. Also Instabug shows many images in the UI:

![image](https://user-images.githubusercontent.com/283886/43597973-7482ca08-9638-11e8-9bac-53be8ba60f9e.png)

Note: this is a nil *image name* not a nil image. So I think this is before the bundle loading part.

**Comment by Kmohamed** [comment]
Ahhh let me check it

**Comment by Kmohamed** [comment]
I am kidding 😅 but your support for Instabug is amazing Thanks a lot @keith  🙏

**Comment by keith** [comment]
Happy to help!

**Comment by Reflejo** [comment]
@Kmohamed You can send him a bunch of pies to Lyft's HQ as a thank you gift anytime. Thought you should know 😉.

**Comment by Kmohamed** [comment]
Sure I will 😂😂

**Comment by keith** [comment]
hahaha

**Comment by keith** [comment]
![2018-08-02 13 39 04](https://user-images.githubusercontent.com/283886/43610109-41515c40-965a-11e8-836b-9cbd598e0ddb.jpg)

Great success! Thanks @Kmohamed! Will submit issues for donuts

**Comment by rebello95** [comment]
Can confirm, donuts were delicious 💯 🍩

**Comment by Kmohamed** [comment]
Glad that you guys like it 😄 Thanks again :🙏

**Comment by Kmohamed** [comment]
@keith we just release a fix for this issue in SDK version 8.0.8

**Comment by keith** [comment]
Thanks!
